packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -154,4 +154,73 @@ describe('Cypress In Cypress E2E', { viewportWidth: 1500, defaultCommandTimeout:
     cy.get('[data-model-state="failed"]').should('contain', 'renders the blank page')
     cy.percySnapshot()
   })
+
+  it('shows automation disconnected warning', () => {
```

Comment:
I wonder if we want to have these two very similar tests in the same file? We could even use the same test for both CT and E2E, with something like:

```js
;['component, 'e2e'].forEach(testingType => {
  it(`handles automation disconnects in ${testingType}`, () => {
    cy.scaffoldProject('cypress-in-cypress')
    cy.findBrowsers()
    cy.openProject('cypress-in-cypress')
    cy.startAppServer(testingType)
  })
})
```

I somewhat foresee more "it should work the same in CT and E2E"-like tests in the future, so we might want a pattern for this, to reduce duplication and decrease the chance of a testing type specific bug.